### DISCUSSION TITLE
Change footnote symbol used for epub

### DISCUSSION THREAD

**Discussion by stephanedebove** [jgm/pandoc#9355] [Q&A]
I am using pandoc to convert a .tex file to an epub.

I have both footnotes and citations in my .tex file. The citations are already using superscripted numbers, so I would like to change the default markers pandoc uses for footnotes (also numbers) to capital letters.

In my .tex file I have added these lines to obtain the correct behavior:

```
\usepackage{alphalph}
\makeatletter
% Redefine the footnote mark to use the alphalph representation
\renewcommand*{\thefootnote}{%
    \AlphAlph{\value{footnote}}%
}
\makeatother
```

but this is not taken into account by pandoc when converting. Is it possible to change the symbols pandoc uses for footnotes in an epub?

MWE:

```
\documentclass{article}

\usepackage{alphalph}
\makeatletter
% Redefine the footnote mark to use the alphalph representation
\renewcommand*{\thefootnote}{%
    \AlphAlph{\value{footnote}}%
}
\makeatother
\begin{document}
\title{Test Document}
\author{Author Name}
\maketitle
\section{Introduction}
This is a test citation\footnote{This is my footnote}.
Lorem ipsum dolor sit amet, consectetur adipiscing elit\footnote{This is my second footnote}.
\end{document}
```
` pandoc .\test.tex -o test.epub 
`

**Comment by jgm** [comment]
There's no easy way to do this (short of modifying pandoc's source code).
Perhaps you could do it with some JavaScript.

A simpler way to make the distinction would be to add some distinctive CSS styling so that the numbers for citations look different from the numbers for footnotes. See the .xhtml files inside the epub container (which you can unzip using 'unzip') to see what the HTML looks like for each.
